### PR TITLE
feat(lang): En lang update

### PR DESCRIPTION
**Pull request by Flemmli97**
### What this PR does 📖

- Removes unused lang keys in en lang file
- Adds a python script that scans the whole app for unused language keys. Script has extensive comments

### CODE REVIEW

**Comment by stavares843** [comment]
very cool 🔨

**Review by stavares843 — APPROVED** [review]

File: utils/lang_checker.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import argparse
+
+# Determines unused lang keys
+# Run by going to the root dir and using `python ./utils/lang_checker.py`
+# It will print out all unused lang keys
+# Can also automatically remove unused lang keys
+
+parser = argparse.ArgumentParser("lang_checker")
+parser.add_argument("--lang", help="The lang file to check", 
+                    type=str, default='./common/locales/en-US/main.ftl', required=False)
```
Comment (stavares843) [inline utils/lang_checker.py:11]: could we update the script to also remove unused keys on the other locales? example spanish, portuguese etc

File: common/locales/en-US/main.ftl [hunk 2]
```diff
@@ -278,15 +253,14 @@ settings-accessibility = Accessibility Settings
...
-    .notifications-description = Enable notifications for incoming calls, messages, and more.
     .enabled = Enabled
     .enabled-description = Enable notifications for incoming calls, messages, and more.
     .grant-permissions = Grant Permissions
     .grant-permissions-description = Grant permissions to receive notifications.
     .friends-description = Enable notifications for friend requests.
     .messages-description = Enable notifications for new messages.
     .settings-description = Enable notifications for updates and important alerts.
-
+
```
Comment (stavares843) [inline common/locales/en-US/main.ftl]: do we need this spacing? 🔨

File: utils/lang_checker.py (hunk 1 above)
Comment (Flemmli97) [reply utils/lang_checker.py:11]: you can already define which lang file to use. unless you mean it should check all files by default?

**Review by phillsatellite — APPROVED** [review]

**Review by dariusc93 — APPROVED** [review]

**Pull request merged**